vdb-bench-assembly/plugins/vdb-bench-dataservice/content/css/styles.less

```diff
@@ -706,3 +706,16 @@ button[id^='dropdownKebabRight_'] {
     position: relative;
     top: -4em;
 }
+
+.dsb-wizard-container .modal-header {
+    padding: 0px 15px;
+}
+
+.dsb-wizard-container .wizard-pf-steps-indicator {
```

Comment:
Yes they are included in the standard class.  I was unsure of the best way to 'override' them.  What I have works, but looking for feedback if what I did is "correct".  I can add !important if needed